lambda_src/drivers/process_https.py

```diff
@@ -17,7 +18,7 @@ def make_basic_header(auth):
 
 
 def parse_header_dict(value):
-    return {k: decrypt_if_encrypted(v) for k, v in parse_qsl(value)}
+    return {k: v for k, v in parse_qsl(value)}
```

Comment:
rebase on master?

Reply:
Had already rebased this branch actually, the PR just seems confused like you said, a new PR on this branch doesn't contain this change.